### ISSUE TITLE
Multisig transaction signing error with Ledger Nano S on Mac

### ISSUE THREAD

**Issue by Stonepapa** [sparrowwallet/sparrow#1241]
### Version

- M1 Mac @Sonoma 14.1.1
- Ledger Nano S
- Sparrow 1.8.1

### Description

When we attempt to sign a PSBT for a 2-of-3 native segwit multisignature output, it fails.

Error messages:

`
2024-01-23 23:52:36,523 ERROR [JavaFX Application Thread] c.s.s.c.DevicePane [null:-1] Signing Error: ('0x6a82', 'Error in <BitcoinInsType.REGISTER_WALLET: 2> command', '')
com.sparrowwallet.sparrow.io.SignTransactionException: ('0x6a82', 'Error in <BitcoinInsType.REGISTER_WALLET: 2> command', '')
	at com.sparrowwallet.sparrow@1.8.1/com.sparrowwallet.sparrow.io.Hwi.signPSBT(Unknown Source)
	at com.sparrowwallet.sparrow@1.8.1/com.sparrowwallet.sparrow.io.Hwi$SignPSBTService$1.call(Unknown Source)
	at com.sparrowwallet.sparrow@1.8.1/com.sparrowwallet.sparrow.io.Hwi$SignPSBTService$1.call(Unknown Source)
	at javafx.graphics@18/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at javafx.graphics@18/javafx.concurrent.Service.lambda$executeTask$6(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at javafx.graphics@18/javafx.concurrent.Service.lambda$executeTask$7(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
`

Sign message steps:

1. Open Sparrow, and create transfer tx.
2. Sign PSBT, connect, scan and unlock Ledger, use Ledger to sign.
3. Get error message.
<img width="499" alt="image" src="https://github.com/sparrowwallet/sparrow/assets/125890624/0675fe44-2a15-407b-bbb9-cbea32f3ed1f">

Both creating a new PSBT and signing an exist PSBT, we got the error message.

**Comment by craigraw** [comment]
Discussed in the Telegram group - the Nano S is likely out of memory to store the wallet configuration.

**Comment by Stonepapa** [comment]
I have already deleted all other wallets. There are 80kb free memory. 
But when I signed multisig tx by Ledger Nano S in Sparrow, I got the same error messages.
<img width="752" alt="image" src="https://github.com/sparrowwallet/sparrow/assets/125890624/7e818ebe-a345-4483-b849-0a1f6de0ab01">

I think the problem is not out of memory to store.

By the way, it works well when I use Ledger Nano S as a single signature wallet to sign tx.
